Gemfile

```diff
@@ -58,7 +58,7 @@ gem 'state_machines-activerecord'
 gem 'stringex', '~> 2.8.5', require: false
 
 gem 'paypal-sdk-merchant', '1.117.2'
-gem 'stripe'
+gem 'stripe', '~> 13'
```

Comment:
Wouldn't this prevent dependabot from upgrading to higher version ? or do we have a reason to pin it to version 13  ?